From b336d7246a324e8973bc449cb35db40b1627be47 Mon Sep 17 00:00:00 2001
From: Serge Hallyn <serge.hallyn@ubuntu.com>
Date: Thu, 12 Sep 2013 11:17:11 -0500
Subject: [PATCH] ignore ability to init /lxc-monitord.log

We may long-term want to instead decide on a convention under
/var/log, but for now just ignore it.  This will only happen
if lxcpath is read-only.

Signed-off-by: Serge Hallyn <serge.hallyn@ubuntu.com>
---
 src/lxc/lxc_monitord.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lxc/lxc_monitord.c b/src/lxc/lxc_monitord.c
index fda6cf4..4049345 100644
--- a/src/lxc/lxc_monitord.c
+++ b/src/lxc/lxc_monitord.c
@@ -342,7 +342,7 @@ int main(int argc, char *argv[])
 
 	ret = lxc_log_init(NULL, logpath, "NOTICE", "lxc-monitord", 0, lxcpath);
 	if (ret)
-		return ret;
+		INFO("Failed to open log file %s, log will be lost", lxcpath);
 
 	pipefd = atoi(argv[2]);
 
-- 
1.8.3.2

